spring-boot-project/spring-boot-docs/src/docs/asciidoc/production-ready-features.adoc

```diff
@@ -1741,17 +1744,63 @@ To export metrics to {micrometer-registry-docs}/dynatrace[Dynatrace], your API t
 	        uri: "YOUR_URI"
 ----
 
-You can also change the interval at which metrics are sent to Dynatrace:
```

Comment:
I think it would make sense to leave the example setting "step" in for v1 so we don't alter the existing one.